src/components/VInput/VInput.vue

```diff
@@ -36,6 +36,8 @@
         <span :class="['vts-input__text', classes.text]">
           {{ option.label }}
         </span>
+
+        <slot name="label" />
```

Comment:
Did you test this slot here? Feels like it would repeat the same label for every radio

Reply:
Which it defiantly would 🤦  - I've moved it to inside the `<legend>`